lib/ConstExtract/ConstExtract.cpp

```diff
@@ -207,33 +248,52 @@ gatherConstValuesForPrimary(const std::unordered_set<std::string> &Protocols,
   return Result;
 }
 
-std::string toString(const CompileTimeValue *Value) {
-  switch (Value->getKind()) {
-    case CompileTimeValue::RawLiteral:
-      return cast<RawLiteralValue>(Value)->getValue();
-    case CompileTimeValue::InitCall:
-      // TODO
-    case CompileTimeValue::Builder:
-      // TODO
-    case CompileTimeValue::Dictionary:
-      // TODO
-    case CompileTimeValue::Runtime:
-      return "Unknown";
+void writeProperty(llvm::json::OStream &JSON,
+                   const ConstValueTypePropertyInfo PropertyInfo) {
+  const auto *PropertyDecl = PropertyInfo.VarDecl;
```

Comment:
nit: I'd favor naming as `decl` rather than `PropertyDecl`. Since this is within a function to write JSON for a property argument, it's kind of implicit, and would be a little more concise.